build.gradle

```diff
@@ -57,23 +57,33 @@ configurations {
 }
 
 dependencies {
-    implementation 'org.apache.kafka:kafka-clients:2.3.1'
-    implementation 'org.apache.kafka:kafka-streams:2.3.1'
+    implementation ('org.apache.kafka:kafka-clients:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    implementation ('org.apache.kafka:kafka-streams:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
     implementation 'org.apache.avro:avro:1.9.0'
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'org.apache.commons:commons-configuration2:2.7'
     implementation 'commons-lang:commons-lang:2.6'
     implementation 'org.slf4j:slf4j-api:1.7.30'
     implementation 'net.sf.jopt-simple:jopt-simple:5.0.4'
-    testImplementation 'org.apache.kafka:kafka_2.12:2.3.1'
+    testImplementation('org.apache.kafka:kafka_2.12:2.3.1') {
+        exclude group: 'org.scala-lang', module: 'scala-reflect'
+    }
+    testImplementation group: 'org.scala-lang', name: 'scala-reflect', version: '2.12.11'
```

Comment:
latest release is 2.13.2 . Why 2.12.11 ?

Reply:
Because new version needs another version of kafka.
The other way is to change kafka, or get a compilation error.

Reply:
In case of change we need to decide between kafka 2.4.1 or kafka 2.5.0.

Reply:
Got this compilation error: /src/test/java/com/opendxl/databus/util/Topic.java:65: error: cannot access Serializable
                new TopicCommand.ZookeeperTopicService(zkUtils).createTopic(opts);
                                                               ^
  class file for scala.Serializable not found

Reply:
So las compatible version is 2.12.11 released on march 2020.

Reply:
Discussed and finally will be use 2.12.11